<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassifierSharkRandomForestTrain</class>
 <widget class="QWidget" name="ClassifierSharkRandomForestTrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Comma-separated label values selection:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selectedLabels">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="selectSlices">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Select slices</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="bottomSliceLimitLabel">
         <property name="text">
          <string>Bottom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bottomSliceLimit">
         <property name="text">
          <string>88</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="topSliceLimitLabel">
         <property name="text">
          <string>Top</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="topSliceLimit">
         <property name="text">
          <string>89</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="setAutomatically">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Set Automatically</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="randAttrLowLabel">
             <property name="text">
              <string>Random attr. per node low</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="randAttrLow">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="randAttrHighLabel">
             <property name="text">
              <string>Random attr. per node high</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="randAttrHigh">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="randAttrStepsLabel">
             <property name="text">
              <string>Steps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="randAttrSteps">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="QLabel" name="numTreesLowLabel">
             <property name="text">
              <string>Number of trees low</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="numTreesLow">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="QLabel" name="numTreesHighLabel">
             <property name="text">
              <string>Number of trees high</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="numTreesHigh">
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_18">
           <item>
            <widget class="QLabel" name="numTreesStepsLabel">
             <property name="text">
              <string>Steps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="numTreesSteps">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_20">
           <item>
            <widget class="QLabel" name="nodeSizeLowLabel">
             <property name="text">
              <string>Node size low</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nodeSizeLow">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item>
            <widget class="QLabel" name="nodeSizeHighLabel">
             <property name="text">
              <string>Node size high</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nodeSizeHigh">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_21">
           <item>
            <widget class="QLabel" name="nodeSizeStepsLabel">
             <property name="text">
              <string>Steps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nodeSizeSteps">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_25">
           <item>
            <widget class="QLabel" name="oobLowLabel">
             <property name="text">
              <string>Out-of-bag ratio low</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="oobLow">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_26">
           <item>
            <widget class="QLabel" name="oobHighLabel">
             <property name="text">
              <string>Out-of-bag ratio high</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="oobHigh">
             <property name="text">
              <string>0.8</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_27">
           <item>
            <widget class="QLabel" name="oobStepsLabel">
             <property name="text">
              <string>Steps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="oobSteps">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="randAttrLabel">
             <property name="text">
              <string>Random attributes per node</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="numTreesLabel">
             <property name="text">
              <string>Number of trees</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nodeSizeLabel">
             <property name="text">
              <string>Node size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="oobLabel">
             <property name="text">
              <string>Out-of-bag ratio</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="randAttr">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="numTrees">
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nodeSize">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="oob">
             <property name="text">
              <string>0.6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
